### PR TITLE
fix PR template to mention recommendatory nature of CC and DS

### PR DESCRIPTION
**Pull request by shcheklein**
* [x] Have you followed the guidelines in our
      [Contributing document](https://dvc.org/doc/user-guide/contributing/core)?

* [x] Does your PR affect documented changes or does it add new functionality
      that should be documented? If yes, have you created a PR for
      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
      least opened an issue for it? If so, please add a link to it.

-----

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
 
-* [ ] Does your PR affect documented changes or does it add new functionality
-      that should be documented? If yes, have you created a PR for
-      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
-      least opened an issue for it? If so, please add a link to it.
+* [ ] 📖 This PR does _not_ require [documentation](https://dvc.org/doc) updates
+      or modifications. If it _does_, have you created a PR for
+      [dvc.org](https://github.com/iterative/dvc.org) or at least opened an
+      issue for it? If so, please add a link to it and after that check this
+      box.
```
Comment (jorgeorpinel) [inline .github/PULL_REQUEST_TEMPLATE.md]: I get the idea but checking a _not_ item may be confusing, how about:

If this PR require [documentation](https://dvc.org/doc) updates, have you created a separate PR in [dvc.org](https://github.com/iterative/dvc.org), or at least opened an issue there about it? **Please link to either one in the description of this PR.**

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
+
+* [ ] ❌ Have you checked DeepSource, CodeClimate, and other sanity checks
+      below? Their findings are considered recommendatory. We don't expect all
```
Comment (jorgeorpinel) [inline .github/PULL_REQUEST_TEMPLATE.md]: "We consider their findings recommendatory and don't expect everything to be addresses. Please review them carefully and fix those that actually improve code or fix bugs."

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
+      of them to be fixed. Please, read them carefully and fix those that
+      actually improve code and/or fix bugs.
+
+Thank you for the contribution - we'll try to review it as soon as possible 🙏.
```
Comment (jorgeorpinel) [inline .github/PULL_REQUEST_TEMPLATE.md]: "Thank you for the contribution! 🙏  We'll try to review it as soon as possible."

Comment (jorgeorpinel) [reply .github/PULL_REQUEST_TEMPLATE.md]: Idk about this one. The emoji also looks good at the end but I use period before emojis: "possible. 🙏"

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (shcheklein) [reply .github/PULL_REQUEST_TEMPLATE.md]: that's was the whole point of doing this PR :) It was not clear if one should check this box if documentation updates are not required. If one does not check it, PR looks incomplete and reviewers should think why is it not checked.

Comment (shcheklein) [reply .github/PULL_REQUEST_TEMPLATE.md]: there should be a clear intention to click this and be done with it if doc modification is not required - this is def majority of cases

Comment (jorgeorpinel) [reply .github/PULL_REQUEST_TEMPLATE.md]: I thought the main point was to "mention recommendatory nature of CC and DS" 😋

> With the current (positive) template if one does not check it, PR looks incomplete

Good point. Maybe just be even more direct (and incorporate some of my wording) like:

Check this box if this PR **does not** require documentation updates, or if it does **and** you have created a **separate PR** in dvc.org with such updates (or at least opened an issue about it in that repo). Please link to your PR (or issue) in the documentation repository below.

Comment (shcheklein) [reply .github/PULL_REQUEST_TEMPLATE.md]: > I thought the main point was to "mention recommendatory nature of CC and DS" 

yeah, kinda ... hidden agenda :)

Comment (shcheklein) [reply .github/PULL_REQUEST_TEMPLATE.md]: done! took your wording, thanks!

**Comment by efiop** [comment]
Thanks!

**Pull request merged**